### ISSUE TITLE
Fork "development": RIMA-Backend doesn't load model files from $MODEL_DIR

### ISSUE THREAD

**Issue by ralf-berger** [ude-soco/RIMA#317]
Information received via email:
> The files that need to be downloaded are mentioned in the "Installation Guide for RIMA-Backend" (steps 7 to 9). Just so you know – those files are required to be placed accordingly to the location mentioned. Let me know whether we need to update it using "os.path.abspath" to let the application find the files

**Comment by ralf-berger** [comment]
"Installation Guide for RIMA-Backend" referrs to the project [README](https://github.com/ude-soco/RIMA/blob/development/README.md#step-1-installation-guide-for-rima-backend).

**Comment by ralf-berger** [comment]
Stuff required for models to work:

- [x] Environment variables for model URLs and paths declared, see `./model-downloader/README.md`
- [x] Downloads being performed, see `./model-downloader/bin/download`
- [ ] Models being loaded from configured locations by RIMA-Backend

Model data (names used, data file/directory path, environment variable names):

- [x] Elmo, `./elmo/elmo.hdf5`, `ELMO_WEIGHT_*`
- [x] StanfordCoreNLP/CoreNLP `./stanford-corenlp/`, `CORE_NLP_*`
- [x] MSmarco/MS MARCO `./msmarco/`, `MS_MARCO_*`

**Comment by ralf-berger** [comment]
Checked that downloads of the models mentioned in the README are being performed:

```sh
$ rm -rf tmp && docker compose -f docker-compose.yml -f docker-compose-dev.yml \
    run model-downloader && ls tmp/model
# => elmo  keyword_extractor  msmarco  nltk  stanford-corenlp
```

**Comment by ralf-berger** [comment]
@shoebjoarder Yes, the application needs to be updated so that it looks for model files relative to `$MODEL_DIR`.

**Comment by ralf-berger** [comment]
First model file (`ELMO_WEIGHT`) seems to work:

```
Ensuring model data is present ...
Mirroring data from Model Server (http://sc-rima-model-server:8080) to /tmp/models ...
223 model files available
Migrating database ...
[...]
[allennlp.commands.elmo:174] elmo 12 139951431134080 Initializing ELMo.
SIF_MODEL_FILE_PATH not set; skip loading Transformer model
```

The next issue seems to be a dependency on the Java JVM:

```
Traceback (most recent call last):
  [...]
  File "/home/app/interests/Keyword_Extractor/Algorithms/embedding_based/data_model.py", line 13, in <module>
    SIF_RANK = SifRank.KeyphraseExtractor(embedding_model=sif_model)
  [...]
  File "/home/app/.venv/lib/python3.7/site-packages/stanfordcorenlp/corenlp.py", line 46, in __init__
    if not subprocess.call(['java', '-version'], stdout=subprocess.PIPE, stderr=subprocess.STDOUT) == 0:
  [...]
FileNotFoundError: [Errno 2] No such file or directory: 'java': 'java'
Exception ignored in: <function KeyphraseExtractor.__del__ at 0x7f48556b45f0>
Traceback (most recent call last):
  File "/home/app/interests/Keyword_Extractor/Algorithms/embedding_based/sifrank/SifRank.py", line 28, in __del__
    self.tagger_model.close()
AttributeError: 'KeyphraseExtractor' object has no attribute 'tagger_model'
```

**Comment by ralf-berger** [comment]
Next issue during startup:

```
[...]
File "/home/app/interests/Keyword_Extractor/Algorithms/embedding_based/data_model.py", line 2, in <module>
from flair.embeddings import TransformerWordEmbeddings
[...]
File "/home/app/.venv/lib/python3.7/site-packages/gensim/models/fasttext.py", line 496, in FastText
@gensim.utils.deprecated("Attribute will be removed in 4.0.0, use wv.min_n instead")
AttributeError: module 'gensim' has no attribute 'utils'

During handling of the above exception, another exception occurred: [...]
```

No idea is this is a model file or a Python dependency thing.

**Comment by ralf-berger** [comment]
Previous error seemingly solved by 126fa45.

Next issue during startup:

```
[...]
  File "/home/app/interests/Keyword_Extractor/Algorithms/embedding_based/data_model.py", line 13, in <module>
    SIF_RANK = SifRank.KeyphraseExtractor(embedding_model=sif_model)
  File "/home/app/interests/Keyword_Extractor/Algorithms/embedding_based/sifrank/SifRank.py", line 22, in __init__
    self.tagger_model = get_POSTagger(self.tagger)
  File "/home/app/interests/Keyword_Extractor/Algorithms/embedding_based/sifrank/utils.py", line 45, in get_POSTagger
    return StanfordCoreNLPTagger(StanfordCoreNLP(settings.STANFORDCORENLP, quiet=True))
  File "/home/app/.venv/lib/python3.7/site-packages/stanfordcorenlp/corenlp.py", line 74, in __init__
    self.lang) + ' not exists. You should download and place it in the ' + directory + ' first.')
OSError: stanford-corenlp-x.x.x-models.jar not exists. You should download and place it in the /tmp/models/stanford-corenlp/ first.
Exception ignored in: <function KeyphraseExtractor.__del__ at 0x7f6f0d5b75f0>
Traceback (most recent call last):
  File "/home/app/interests/Keyword_Extractor/Algorithms/embedding_based/sifrank/SifRank.py", line 28, in __del__
    self.tagger_model.close()
AttributeError: 'KeyphraseExtractor' object has no attribute 'tagger_model'
```

**Comment by ralf-berger** [comment]
Resolved by extracting model archives into correctly directories: 52b6cd2, e889a54
